var/spack/repos/builtin/packages/py-bluepyopt/package.py

```diff
@@ -12,6 +12,8 @@ class PyBluepyopt(PythonPackage):
     homepage = "https://github.com/BlueBrain/BluePyOpt"
     url = "https://pypi.io/packages/source/b/bluepyopt/bluepyopt-1.9.27.tar.gz"
 
+    version('CMA_clean', branch='CMA_clean')
```

Comment:
Needs a `git` url? There's only the PyPI one specified!

Reply:
ah wait, I thought I had removed this change, we don't need this version anymore, let me remove that.